master/reference/calicoctl/commands/index.md

```diff
@@ -53,7 +52,54 @@ organized by top level command.
 -  [calicoctl apply]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/apply)
 -  [calicoctl delete]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/delete)
 -  [calicoctl get]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/get)
--  [calicoctl config]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/config)
 -  [calicoctl ipam]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/ipam)
 -  [calicoctl node]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/node)
 -  [calicoctl version]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/version)
+
+## Modifying low-level component configurations
```

Comment:
Why is this in the command index?

Reply:
These are instructions specific to a particular resource, each of which has its own document detailing its use.

I think these should go in the felixconfiguration and bgpconfiguration docs